### ISSUE TITLE
Crash report: "missing field `jquery`"

### ISSUE THREAD

**Issue by NavinF** [andrewbanchich/shreddit#14]
```
RUST_BACKTRACE=1 SHREDDIT_DRY_RUN=false ./shreddit-linux
<snipped output>
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Decode, source: Error("missing field `jquery`", line: 1, column: 50) }', src/things.rs:206:14
stack backtrace:
   0: rust_begin_unwind
             at /rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/std/src/panicking.rs:575:5
   1: core::panicking::panic_fmt
             at /rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/core/src/panicking.rs:64:14
   2: core::result::unwrap_failed
             at /rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/core/src/result.rs:1791:5
   3: shreddit::things::Thing::edit::{{closure}}::{{closure}}
   4: <tracing::instrument::Instrumented<T> as core::future::future::Future>::poll
   5: shreddit::things::Thing::shred::{{closure}}
   6: tokio::runtime::park::CachedParkThread::block_on
   7: tokio::runtime::scheduler::multi_thread::MultiThread::block_on
   8: tokio::runtime::runtime::Runtime::block_on
   9: shreddit::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.

```

```
./shreddit-linux --version
shreddit 0.4.0
```

**Comment by andrewbanchich** [comment]
Could you try running the [0.4.1 release](https://github.com/andrewbanchich/shreddit/releases/tag/v0.4.1)? That should surface the error you're seeing.

**Comment by andrewbanchich** [comment]
Actually, I see this too. It seems to only occur for posts, not comments. Looking into this now.

**Comment by andrewbanchich** [comment]
This is fixed and will be in the `0.4.2` release. Thanks for the report!

**Comment by NavinF** [comment]
Thanks, confirmed it works now :)
